lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -133,6 +133,12 @@ sub dead {
     $self->t_updated < $dt;
 }
 
+sub get_websocket_api_version {
+    my ($self) = @_;
+
+    return $self->get_property('WEBSOCKET_API_VERSION');
```

Comment:
We get this information too late, i think we shoud instead exchange this in a handshake sequence, not in the worker_status timer (this would also explain all scheduler_full test fails)

Reply:
@mudler 
This code is just wrapping over the get_property.

Reply:
Yes it is :) - it was to warn you that when you request this information in the scheduler , you have to be sure to have it. Previously you was getting this information by the worker_status timer, and not in the registration.